Cargo.toml

```diff
@@ -39,7 +39,7 @@ rstest = "0.16.0"
 tonic-build = { version = "0.8.0", optional = true }
 
 [features]
-default = ["sample_kbc", "rust-crypto"]
+default = ["sample_kbc", "rust-crypto", "cc_kbc"]
```

Comment:
Is this just for testing?

Note how other platforms have an additional feature to build their attester. You may want to add something similar.

Reply:
yes, this will be reverted I think.